src/components/MapBox.vue

```diff
@@ -127,6 +131,10 @@ export default {
 
       // Once map is loaded, zoom in a bit more so that the map neatly fills the screen
       map.fitBounds([[-125.3321, 23.8991], [-65.7421, 49.4325]]);
+      //set timeout to make sure the fitbounds is completely done before fadeaway
```

Comment:
Do we need a whole second here?

Reply:
lol I didn't think it was bad, but can see what a shorter time span is like.